add_demo_src/add_demo.py

```diff
@@ -0,0 +1,36 @@
+from exceptions import DuplicateNameOrKeyException
+
+import json
+
+def add(new_demo):
+    try:
+        check_for_duplicates(new_demo.name, new_demo.key)
+        insert_demo(new_demo)
+    except DuplicateNameOrKeyException as e:
+        print(f"DuplicateNameOrKeyException: {e.message}")
+
+    return
+
+def get_existing_demos():
+    with open("../demo_list.json") as json_file:
+        data = json.load(json_file)
+        
+        return data
```

Comment:
This might be shorter as
```py
def get_existing_demos():
  return json.load(open('../demo_list.json'))
```

Reply:
It is shorter, but I find the separate lines a bit easier to read and debug since `data` has its own variable. I'm not really much of a scripting programmer though 😅